pkg/schema/schema.go

```diff
@@ -167,6 +181,30 @@ func newCollectionItemValueType(collectionItemValue interface{}, position *filep
 	return nil, fmt.Errorf("Collection item type did not match any known types")
 }
 
+func processTypeAnnotation(templateAnnotations template.NodeAnnotations) (bool, error) {
+	if ann, anyType := templateAnnotations[AnnotationSchemaType]; anyType {
+		kwargName, err := core.NewStarlarkValue(ann.Kwargs[0][0]).AsString()
+		if err != nil {
+			return false, err
+		}
+		if kwargName != "any" {
+			return false, fmt.Errorf("unknown '%v' annotation keyword argument '%v'. Supported kwargs are 'any'", AnnotationSchemaType, kwargName)
+		}
+		kwargBool, err := core.NewStarlarkValue(ann.Kwargs[0][1]).AsBool()
+		if kwargBool == false && err != nil {
+			kwargBool := core.NewStarlarkValue(ann.Kwargs[0][1]).AsGoValue()
```

Comment:
As a stylistic thing I would probably assign this to a new variable rather than reassign it to `kwargBool ` since they are different types and `kwargBool` is used later on as well.

Reply:
+1.

While we're talkin' names, here. At this point, I think we're certain of the _role_ of the second kwarg: it's whether `any` type is allowed. Surfacing the role into the name (a form of "revealing intention") can make the code more readable. 👍🏼